<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConjugationWidget</class>
 <widget class="QWidget" name="ConjugationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>535</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>-</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="layout">
     <property name="title">
      <string>Conjugation Forms</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="singularFirstPersonLabel">
        <property name="text">
         <string>First</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>singularFirstPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="singularFirstPersonLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="singularSecondPersonLabel">
        <property name="text">
         <string>Second</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>singularSecondPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="singularSecondPersonLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="singularThirdMalePersonLabel">
        <property name="text">
         <string>Third Male</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>singularThirdMalePersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="singularThirdMalePersonLineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="singularThirdFemalePersonLabel">
        <property name="text">
         <string>Third Female</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>singularThirdFemalePersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="singularThirdFemalePersonLineEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="singularThirdNeutralPersonLabel">
        <property name="text">
         <string>Third Neutral</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>singularThirdNeutralPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLineEdit" name="singularThirdNeutralPersonLineEdit"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="dualFirstPersonLabel">
        <property name="text">
         <string>First</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>dualFirstPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="dualFirstPersonLineEdit"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="dualSecondPersonLabel">
        <property name="text">
         <string>Second</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>dualSecondPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QLineEdit" name="dualSecondPersonLineEdit"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="dualThirdMalePersonLabel">
        <property name="text">
         <string>Third Male</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>dualThirdMalePersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="dualThirdFemalePersonLabel">
        <property name="text">
         <string>Third Female</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>dualThirdFemalePersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="2">
       <widget class="QLineEdit" name="dualThirdFemalePersonLineEdit"/>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="dualThirdNeutralPersonLabel">
        <property name="text">
         <string>Third Neutral</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>dualThirdNeutralPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="1" colspan="2">
       <widget class="QLineEdit" name="dualThirdNeutralPersonLineEdit"/>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="pluralFirstPersonLabel">
        <property name="text">
         <string>First</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>pluralFirstPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="15" column="1" colspan="2">
       <widget class="QLineEdit" name="pluralFirstPersonLineEdit"/>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="pluralSecondPersonLabel">
        <property name="text">
         <string>Second</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>pluralSecondPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="16" column="1" colspan="2">
       <widget class="QLineEdit" name="pluralSecondPersonLineEdit"/>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="pluralThirdMalePersonLabel">
        <property name="text">
         <string>Third Male</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>pluralThirdMalePersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="2">
       <widget class="QLineEdit" name="dualThirdMalePersonLineEdit"/>
      </item>
      <item row="17" column="1" colspan="2">
       <widget class="QLineEdit" name="pluralThirdMalePersonLineEdit"/>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="pluralThirdFemalePersonLabel">
        <property name="text">
         <string>Third Female</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>pluralThirdFemalePersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="18" column="1" colspan="2">
       <widget class="QLineEdit" name="pluralThirdFemalePersonLineEdit"/>
      </item>
      <item row="19" column="1" colspan="2">
       <widget class="QLineEdit" name="pluralThirdNeutralPersonLineEdit"/>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="pluralThirdNeutralPersonLabel">
        <property name="text">
         <string>Third Neutral</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>pluralThirdNeutralPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="tenseComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="nextTenseButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ne&amp;xt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="tenselabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Tense:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>tenseComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="singularLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Singular</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>singularFirstPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="dualLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Dual</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>dualFirstPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLabel" name="pluralLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Plural</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>pluralFirstPersonLineEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
